Description: Disable broken test
 This patch disables test_change_password_shows_message_on_login_page which
 upstream consoder broken with latest django-openstack-auth.
 .
     Disable broken unit test (related to "Change Password")

    The new django_openstack_auth 1.1.6 library surfaced a defect in this
    test, whereby the test client does not properly log out the user. This
    only affects the test, the functionality otherwise works fine.

    This temporarily disables the test in order to repair the gate, while
    waiting to replace it with a new unit test more reduced in scope and/or
    an integration test.
Author: Julie Pichon <jpichon@redhat.com>
Origin: upstream, https://review.openstack.org/#/c/101857/
Bug-Ubuntu: https://launchpad.net/bugs/1333144
Last-Update: 2014-08-05

--- horizon-2014.1.1.orig/openstack_dashboard/dashboards/settings/password/tests.py
+++ horizon-2014.1.1/openstack_dashboard/dashboards/settings/password/tests.py
@@ -52,19 +52,3 @@
         res = self.client.post(INDEX_URL, formData)
 
         self.assertFormError(res, "form", None, ['Passwords do not match.'])
-
-    @test.create_stubs({api.keystone: ('user_update_own_password', )})
-    def test_change_password_shows_message_on_login_page(self):
-        api.keystone.user_update_own_password(IsA(http.HttpRequest),
-                                              'oldpwd',
-                                              'normalpwd').AndReturn(None)
-        self.mox.ReplayAll()
-
-        formData = {'method': 'PasswordForm',
-                    'current_password': 'oldpwd',
-                    'new_password': 'normalpwd',
-                    'confirm_password': 'normalpwd'}
-        res = self.client.post(INDEX_URL, formData, follow=True)
-
-        info_msg = "Password changed. Please log in again to continue."
-        self.assertContains(res, info_msg)
